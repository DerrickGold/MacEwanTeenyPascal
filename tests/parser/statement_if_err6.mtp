begin
	(* Missing parenthesis: right*)
	if (something then
		sample := 2

end
.