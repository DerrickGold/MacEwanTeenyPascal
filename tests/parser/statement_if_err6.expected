Syntax Error: line 3:
Unexpected token (TOK_KEY_THEN, 'then'), likely missing a(n)  
  'TOK_RPAREN' or
Possibly due to malformed or missing:
  Non-terminal: 'Condition'
